rtt/base/DataObjectLockFree.hpp

```diff
@@ -198,34 +237,77 @@ namespace RTT
              * not increment the counter on write_ptr+1). Hence, no
              * locking is needed.
              */
+
+            if (!initialized) {
+                types::TypeInfo *ti = types::Types()->getTypeInfo<DataType>();
```

Comment:
.... I would suggest to at least add a way for the TypeInfo object to tell that the type is RT-safe by itself.

And a lot of Rock users really don't care that much about RT-safety. Having logs and output filled with errors that you really don't care about is really not that great. Could we at least demote this message to Info ? Or have a way to disable it ?